grid/recruitment/create-lead/profile.supr

```diff
@@ -0,0 +1,443 @@
+"""
+Manage Leads
+Create sourced leads in ATS (Applicant Tracking System).
+"""
+name = "recruitment/create-lead"
+version = "1.0.0"
+
+"""
+Create Lead
+Creates a possible candidate either for some specific job or just for ATS.
```

Comment:
```suggestion
Creates a sourced prospect candidate either for some specific job or any job within a company.
```
suggestion: Some ATS accounts are connected with multiple companies, some just one, but this use case should create the lead for just one company.